phy/cluster/manual/session.py

```diff
@@ -201,18 +188,36 @@ def _store_cluster(self,
                               append=True,
                               )
 
-    def store_all_clusters(self, spikes_per_cluster):
+    def is_consistent(self, cluster, spikes):
+        """Return whether the data of a cluster is consistent."""
```

Comment:
You could MD5 both the cluster and the store object, but I imagine that'd take longer!